include/seqan3/range/views/minimiser_hash.hpp

```diff
@@ -0,0 +1,153 @@
+    */
+    constexpr auto operator()(shape const & shape, uint32_t const window_size, uint64_t const seed) const
+    {
+        return seqan3::detail::adaptor_from_functor{*this, shape, window_size, seed};
+    }
+
+    /*!\brief            Call the view's constructor with the underlying view, a seqan3::shape and a window size as
+     *                   argument.
+     * \param[in] urange      The input range to process. Must model std::ranges::viewable_range and the reference type
+     *                        of the range must model seqan3::semialphabet.
+     * \param[in] shape       The seqan3::shape to use for hashing.
+     * \param[in] window_size The windows_size to use.
+     * \param[in] seed        The seed to use.
+     * \throws std::invalid_argument if the size of the shape is greater than the window_size.
+     * \returns               A range of converted elements.
+     */
+    template <std::ranges::range urng_t>
+    constexpr auto operator()(urng_t && urange, shape const & shape, uint32_t const window_size,
+                              uint64_t const seed = 0x8F3F73B5CF1C9ADE) const
+    {
+        static_assert(std::ranges::viewable_range<urng_t>,
+            "The range parameter to views::minimiser_hash cannot be a temporary of a non-view range.");
+        static_assert(std::ranges::forward_range<urng_t>,
+            "The range parameter to views::minimiser_hash must model std::ranges::forward_range.");
+        static_assert(semialphabet<reference_t<urng_t>>,
+            "The range parameter to views::minimiser_hash must be over elements of seqan3::semialphabet.");
+
+        if (shape.size() > window_size)
+            throw std::invalid_argument{"The size of the shape cannot be greater than the window size."};
+
+        return std::forward<urng_t>(urange) | seqan3::views::kmer_hash(shape)
+                                            | std::views::transform([seed] (uint64_t i) { return i ^ seed; })
+                                            | seqan3::views::minimiser(window_size - shape.size() + 1);
+    }
+};
+
+} // namespace seqan3::detail
+
+namespace seqan3::views
+{
+
+/*!\name Alphabet related views
+ * \{
+ */
+
+/*!\brief                    Computes minimisers for a range with a given shape, window size and seed.
+ * \tparam urng_t            The type of the range being processed. See below for requirements. [template parameter is
+ *                           omitted in pipe notation]
+ * \param[in] urange         The range being processed. [parameter is omitted in pipe notation]
+ * \param[in] shape          The seqan3::shape that determines how to compute the hash value.
+ * \param[in] window_size    The window size to use.
+ * \param[in] seed           The seed to use. Default: 0x8F3F73B5CF1C9ADE.
+ * \returns                  A range of `size_t` where each value is the minimiser of the resp. window.
+ *                           See below for the properties of the returned range.
```

Comment:
Just so you know, the general rule is that we keep the indentation level for one type of doc entity, e.g. we align all `\param`, all `\tparam`, etc,

But it's not important and you don't need to change anything, but sometimes it may be easier if you don't have to abide the indentation level of sine long `\param` when you want to do the `\brief` or `\returns`.